pkg/backend/apply.go

```diff
@@ -152,6 +153,12 @@ func confirmBeforeUpdating(kind apitype.UpdateKind, stack Stack,
 
 		choices := []string{string(yes), string(no)}
 
+		// If we have a new plan but didn't start with a plan we can prompt to use the new plan.
+		// If we're in experimental mode we don't add this because "yes" will also use the plan
+		if plan != nil && opts.Engine.Plan == nil && !opts.Engine.Experimental {
+			choices = append(choices, string(yesPlan))
```

Comment:
I haven't tried this out yet, but will this show the new choice under "no"? I thought we want it above "yes" per Luke's suggestion:

```
Do you want to perform this update?  [Use arrows to move, enter to select, type to filter]
  [experimental] yes, using Update Plans (https://pulumi.com/updateplans)
  yes
> no
  details
```

Reply:
Ah yeh can flip that around.